actions/filesystem_deploy_action.go

```diff
@@ -1,3 +1,22 @@
+/*
+FilesystemDeploy Action
+
+Deploy prepared root filesystem to output image. This action must be started
+after 'image-partition' action.
```

Comment:
Hrm, the wording is not ideal here, my reading would be that you have to run this after image-partition which isn't true :p  ``This action requires image-partition to be executed before it`` may be better?

Reply:
done